assets/css/sensei-course-theme/layout.scss

```diff
@@ -35,14 +36,25 @@
 	}
 
 	&__main-content {
-		padding: 32px;
+		padding: --content-padding;
 		margin-left: var(--sidebar-width) !important;
 
 		> * {
 			max-width: 900px;
 			margin-left: auto;
 			margin-right: auto;
 		}
+
+		.alignfull {
```

Comment:
Works well, but not sure the layout file is the best place for this. We could start up a `content.scss`. (And just move the variables to `:root`)

Reply:
I think you're right. I'll move it to new `content.scss` file. 👍

Reply:
On another thought. I think these styles makes sense to be located in `layout.scss`. Because it deals with the layout. I also think it's closely related to styles at lines 42:46. Will leave them here for now.

Reply:
It's not dealing with the layout of the course theme page, like the rest of the file, but with styling an element inside the post content. Sure it needs some information about its context, but it can access that from an exposed variable.

Reply:
I am addressing this in another PR here #4524 . Moved the css related to content into `content.scss`. 👍